Frequently-Asked-Questions.mediawiki

```diff
@@ -80,6 +80,12 @@ GNU/Linux:
 ** <code>~/.local/share/qBittorrent/</code>
 ** <code>~/.local/share/data/qBittorrent/</code> (in case legacy data directory was used)
 
+Linux/Flatpak:
+* preferences
+** <code>~/.var/app/org.qbittorrent.qBittorrent/config/qBittorrent/</code>
+* .torrent files, logs, etc (This is the standard [https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html XDG] data folder):
```

Comment:
Under `Linux/SnapStore:` section the `(This is the standard [https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html XDG] data folder)` part is also wrong.
Would you help strip it too?

Reply:
I made the changes you requested. Thanks for pointing out the mistake with `$XDG_DATA_HOME`. Can't believe I missed that.